account_manager/src/validator/list.rs

```diff
@@ -1,24 +1,28 @@
-use crate::VALIDATOR_DIR_FLAG;
+use account_utils::validator_definitions::ValidatorDefinitions;
 use clap::App;
 use std::path::PathBuf;
-use validator_dir::Manager as ValidatorManager;
 
 pub const CMD: &str = "list";
 
 pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
-    App::new(CMD).about("Lists the names of all validators.")
+    App::new(CMD).about("Lists the public keys of all validators.")
```

Comment:
This is not strictly true, since it lists directory names.

Reply:
It used to... maybe this is a bit sneaky on my part, but I've changed it so that it reads the `validator_definitions.yaml` and prints the pubkeys themselves. I think that's a bit cleaner than the directory name thing we were doing before, and allows us to delete the `validator_dir::Manager` which was only used here.